### PR TITLE
feat[exercise 1]:found character with unknown birthyear

### PR DESCRIPTION
**Pull request by NoamRivlin**
- found a character with an unknown birth year 

inside the findCharacterUnknownBirthDate:
-added a findCharacter variable
-and the function returned the character found inside the variable

Resources:
https://www.youtube.com/watch?v=eBGIQ7ZuuiU

### CODE REVIEW

File: exercises/exercise_1.js [hunk 1]
```diff
@@ -1,9 +1,16 @@
-const characters = require("../data/characters.json");
+const characters = require('../data/characters.json');
 
 /* TODO: return one character with an "unknown" birth year  */
```
Comment (ChrisKlingsporn) [inline exercises/exercise_1.js:4]: Empty lines and redundant comments are to be removed

File: exercises/exercise_1.js [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 
-const findCharacterUnknownBirthDate = () => {
+const findCharacterUnknownBirthDate = (character) => {
   // CODE HERE
```
Comment (ChrisKlingsporn) [inline exercises/exercise_1.js:6]: redundant comment to be removed

File: exercises/exercise_1.js [hunk 3]
```diff
@@ -1,9 +1,16 @@
-const characters = require("../data/characters.json");
+const characters = require('../data/characters.json');
 
 /* TODO: return one character with an "unknown" birth year  */
 
-const findCharacterUnknownBirthDate = () => {
+const findCharacterUnknownBirthDate = (character) => {
```
Comment (ChrisKlingsporn) [inline exercises/exercise_1.js:5]: good use of arrow function. if you're not calling the function outside of the variable, you're allowed to make it an anonymous function as follows: 
```js
const findCharacterUnknownBirthDate = () => {
---dostuffhere---
}
```

File: exercises/exercise_1.js [hunk 4]
```diff
@@ -1,9 +1,16 @@
...
   // CODE HERE
+  //birth_year
+  let findCharacter = [...characters];
```
Comment (ChrisKlingsporn) [inline exercises/exercise_1.js:8]: good choice ripping the array apart. me likey! 🎉🎊

File: exercises/exercise_1.js (hunk 1 above)
Comment (NoamRivlin) [reply exercises/exercise_1.js:4]: done

File: exercises/exercise_1.js (hunk 2 above)
Comment (NoamRivlin) [reply exercises/exercise_1.js:6]: done

File: exercises/exercise_1.js (hunk 3 above)
Comment (NoamRivlin) [reply exercises/exercise_1.js]: right you are

File: exercises/exercise_1.js (hunk 4 above)
Comment (NoamRivlin) [reply exercises/exercise_1.js:7]: 😎

**Comment by NoamRivlin** [comment]
cleaned and refacotred after Chris's review

**Pull request merged**